src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -174,6 +174,21 @@ public GHDiscussion getDiscussion(long discussionNumber) throws IOException {
         return root.createRequest().withUrlPath(api("/members")).toIterable(GHUser[].class, item -> item.wrapUp(root));
     }
 
+    /**
+     * Retrieves the teams that are members of this team.
+     *
+     * @param organization
+     *            the organization that owns this team
+     * @return the paged iterable
+     * @throws IOException
+     *             the io exception
+     */
+    public PagedIterable<GHTeam> listChildTeams(GHOrganization organization) throws IOException {
```

Comment:
The team already knows what organization it is part of, doesn't it?

Reply:
I would have guessed so, but I got an Exception when I tried to use the wrapUp method that didn't take an organization argument.

Is there a better way to solve this?

Reply:
I now see that the organization is stored as a member of the Team.